circleci: failed tests: TestRaftUpdate

The following test appears to have failed:

[#17861](https://circleci.com/gh/cockroachdb/cockroach/17861):

```
I160516 19:31:04.465875 acceptance/replication_test.go:66  found 3 replicas
I160516 19:31:04.465903 acceptance/replication_test.go:74  found 3 replicas
I160516 19:32:04.914642 acceptance/cluster/localcluster.go:658  stopping
I160516 19:32:04.914879 acceptance/cluster/localcluster.go:556  event stream done, resetting...: net/http: request canceled
I160516 19:32:04.914899 acceptance/cluster/localcluster.go:579  events monitor exits
--- FAIL: TestRaftUpdate (65.69s)
	update_test.go:54: acceptance/update_test.go:63: Post https://127.0.0.1:32885/_admin/v1/cluster/freeze: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/update_test.go
+++ b/acceptance/update_test.go
@@ -47,6 +47,7 @@ func postFreeze(c cluster.Cluster, freeze bool, timeout time.Duration) (server.C
 }
 
 func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig) {
+	t.Skip("#6715")
 	minAffected := int64(server.ExpectedInitialRangeCount())
 
 	const long = time.Minute

--- a/acceptance/update_test.go
+++ b/acceptance/update_test.go
@@ -17,6 +17,7 @@
 package acceptance
 
 import (
+	"strings"
 	"testing"
 	"time"
 
@@ -35,9 +36,9 @@ func TestRaftUpdate(t *testing.T) {
 	runTestOnConfigs(t, testRaftUpdateInner)
 }
 
-func postFreeze(c cluster.Cluster, freeze bool) (server.ClusterFreezeResponse, error) {
+func postFreeze(c cluster.Cluster, freeze bool, timeout time.Duration) (server.ClusterFreezeResponse, error) {
 	httpClient := cluster.HTTPClient()
-	httpClient.Timeout = time.Minute
+	httpClient.Timeout = timeout
 
 	var resp server.ClusterFreezeResponse
 	err := postJSON(httpClient, c.URL(0), "/_admin/v1/cluster/freeze",
@@ -48,8 +49,11 @@ func postFreeze(c cluster.Cluster, freeze bool) (server.ClusterFreezeResponse, e
 func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig) {
 	minAffected := int64(server.ExpectedInitialRangeCount())
 
+	const long = time.Minute
+	const short = 10 * time.Second
+
 	mustPost := func(freeze bool) server.ClusterFreezeResponse {
-		reply, err := postFreeze(c, freeze)
+		reply, err := postFreeze(c, freeze, long)
 		if err != nil {
 			t.Fatal(util.ErrorfSkipFrames(1, "%v", err))
 		}
@@ -86,7 +90,13 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	// of the last node receiving the freeze command).
 	// Note that this is the freeze trigger stalling on the Replica, not the
 	// Store-polling mechanism.
-	if reply, err := postFreeze(c, true); !testutils.IsError(err, "timed out waiting for Range|Timeout exceeded while") {
+	acceptErrs := strings.Join([]string{
+		"timed out waiting for Range",
+		"Timeout exceeded while",
+		"connection is closing",
+		"deadline",
+	}, "|")
+	if reply, err := postFreeze(c, true, short); !testutils.IsError(err, acceptErrs) {
 		t.Fatalf("expected timeout, got %v: %v", err, reply)
 	}
 
@@ -127,7 +137,7 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	}
 
 	// Unfreezing again should be a no-op.
-	if reply, err := postFreeze(c, false); err != nil {
+	if reply, err := postFreeze(c, false, long); err != nil {
 		t.Fatal(err)
 	} else if reply.RangesAffected > 0 {
 		t.Fatalf("still %d frozen ranges", reply.RangesAffected)

